pages/search/search.jsx

```diff
@@ -1,21 +1,9 @@
 import React from 'react';
```

Comment:
Simplified this file quite a bit as it's now only used by the `Search page` (and not the `Moderation page`).